gymroutine-mobile/Manager/UserManager.swift

```diff
@@ -53,5 +53,17 @@ class UserManager: ObservableObject {
             createdAt: (data["createdAt"] as? Timestamp)?.dateValue() ?? Date()
         )
     }
+    
+    // follower
+    func fetchFollowersCount(userId: String) async throws -> Int {
+        let snapshot = try await db.collection("Users").document(userId).collection("Followers").getDocuments()
+        return snapshot.documents.count
+    }
+    
+    // following
+    func fetchFollowingCount(userId: String) async throws -> Int {
+        let snapshot = try await db.collection("Users").document(userId).collection("Following").getDocuments()
+        return snapshot.documents.count
+    }
```

Comment:
async throwsは、呼び出し側にエラーを任せられて便利ですが、親側でどのエラーなのかを分析することが困難になるので、do/catchの記載はこのuserManager内で済ますべき事項だと考えます。

```suggestion
    // follower
    func fetchFollowersCount(userId: String) async -> Int {
        do {
            let snapshot = try await db.collection("Users").document(userId).collection("Followers").getDocuments()
            return snapshot.documents.count
        } catch {
            print("[ERROR] フォロワー数の取得に失敗。id: \(userId)")
            return -1
        }
    }
    
    // following
    func fetchFollowingCount(userId: String) async -> Int {
        do {
            let snapshot = try await db.collection("Users").document(userId).collection("Following").getDocuments()
            return snapshot.documents.count
        } catch {
            print("[ERROR] フォロー数の取得に失敗。id: \(userId)")
            return -1
        }
    }
```